package.json

```diff
@@ -37,9 +37,9 @@
     "co-request": "^1.0.0",
     "eventemitter2": "^2.2.1",
     "five-bells-shared": "^23.0.0",
-    "ilp-core": "~13.0.0",
+    "ilp-core": "interledgerjs/ilp-core#mj-release-2.0",
```

Comment:
bumped ilp-core's version to 14.0.0, and ilp-connector's version to 17.0.0. Will merge both so that npm modules get built.